xml/System.Data.Objects.DataClasses/RelatedEnd.xml

```diff
@@ -823,12 +839,30 @@
         <Parameter Name="allowCollection" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <typeparam name="TEntity">To be added.</typeparam>
-        <param name="entity">To be added.</param>
-        <param name="index">To be added.</param>
-        <param name="allowCollection">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <typeparam name="TEntity">The type of the object to validate.</typeparam>
+        <param name="entity">The object being validated.</param>
+        <param name="index">The position of the object being validated in the collection.</param>
+        <param name="allowCollection">Indicates whether the object is a member of a collection.</param>
+        <summary>Determines whether an object can be attached to the local collection or reference.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+The `ValidateEntityForAttach<TEntity>` method is called by Object Services during an attach operation.
+
+]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">
+          When the object being attached is <see langword="null" />.
```

Comment:
Exception information needs to be flush left; this is rendering as a code block.